src/extract.rs

```diff
@@ -55,18 +55,31 @@ pub fn extract_light_occluders(
             &ViewVisibility,
         )>,
     >,
+    camera_query: Query<Entity, With<Camera>>,
 ) {
+    let mut occluders_len = 0;
+
     for (render_entity, light_occluder, transform, view_visibility) in &light_occluders_query {
-        if !view_visibility.get() {
-            continue;
+        if view_visibility.get() {
+            occluders_len += 1;
+
+            commands
+                .entity(render_entity)
+                .insert(ExtractedLightOccluder2d {
+                    half_size: light_occluder.half_size,
+                    center: transform.translation().xy(),
+                });
+        } else {
+            commands
+                .entity(render_entity)
+                .remove::<ExtractedLightOccluder2d>();
         }
+    }
 
-        commands
-            .entity(render_entity)
-            .insert(ExtractedLightOccluder2d {
-                half_size: light_occluder.half_size,
-                center: transform.translation().xy(),
-            });
+    if let Ok(camera) = camera_query.get_single() {
```

Comment:
Here you are adding the `Occluder2dBuffer` to a single camera instance, so the plugin won't work on apps with a multi camera setup (multi monitors, or render targets).

I'd create a new system to add the count of point lights and occluders for each camera separately. You can use the `RenderVisibleEntities` component  for that.